src/attributes.jl

```diff
@@ -143,15 +143,19 @@ Return an attribute `attr` of the model `model`.
 
     get(model::ModelLike, attr::AbstractVariableAttribute, v::VariableIndex)
 
-Return an attribute `attr` of the variable `v` in model `model`.
+If the attribute `attr` is set for the variable `v` in the model `model`, return
+return an empty string if `attr` is [`ConstraintName`](@ref) and `nothing`
```

Comment:
This behavior is better documented on the `ConstraintName` and `VariableName` attributes by saying that they have a default value of "" if not set.